.ci/util/ci_common.sh

```diff
@@ -67,3 +86,29 @@ generate_self_signed_certs() {
                 -subj "/CN=*.${IP_ADDRESS}.nip.io" \
                 -nodes && cat cert.pem key.pem > ca.crt    
 }
+
+minikube_installation() {
+  start_libvirt
+  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+  && chmod +x minikube
+  sudo install minikube /usr/local/bin/ && rm -rf minikube
+}
+
+installEpelRelease() {
+  if yum repolist | grep epel; then
+    printWarn "Epel already installed, skipping instalation."
+  else
+    #excluding mirror1.ci.centos.org 
+    echo "exclude=mirror1.ci.centos.org" >> /etc/yum/pluginconf.d/fastestmirror.conf
```

Comment:
Why are you excluding this mirror here?

Reply:
I discovered few days ago an issues in VM with epel. https://bugs.centos.org/view.php?id=16337

Reply:
Seems like Vipul already closed that issue (few days ago). Can you confirm this is still happening?

Reply:
Hmm, In my Vagrant machine seems the issue is solved.